cfme/automate/service_dialogs.py

```diff
@@ -116,6 +116,19 @@ def __init__(self, label=None, description=None,
         self.box_label = box_label
         self.box_desc = box_desc
 
+    def add_element(self, *element_data):
+        for each_element in element_data:
+            plus_btn({
+                version.LOWEST: "Add a New Element to this Box",
+                "5.3": "Add a new Element to this Box"})
+            sel.wait_for_element(element_form.ele_label)
+            # Workaround to refresh the fields, select other values (text area box and checkbox)and
+            # then select "text box"
+            fill(element_form, {'choose_type': "Text Area Box"})
```

Comment:
@mfalesni is there something here we can do to avoid a workaround like this? Do we know what the issue is?